src/main/java/uk/gov/hmcts/reform/pip/subscription/management/models/Subscription.java

```diff
@@ -0,0 +1,35 @@
+package uk.gov.hmcts.reform.pip.subscription.management.models;
+
+import io.swagger.annotations.ApiModelProperty;
+import lombok.Data;
+
+import java.io.Serializable;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.Table;
+
+@Data
+@Table
+public class Subscription implements Serializable {
+
+    /**
+     * Unique subscription ID.
+     */
+    @Id
+    @GeneratedValue(strategy = GenerationType.AUTO)
+    @ApiModelProperty(hidden = true)
+    private long id;
```

Comment:
maybe this as a UUID?

Reply:
This should now be solved too - switched all occurrences to UUID